src/features/InteractWithContracts/components/FunctionDropdownItem.tsx

```diff
@@ -27,7 +27,7 @@ const Sticker = styled.div<StickerProps>`
   background-color: ${(props) => (props.isRead ? COLORS.SUCCESS_GREEN : COLORS.BLUE_BRIGHT)};
   border-radius: 28px;
   color: ${COLORS.WHITE};
-  padding: 2px 8px;
+  padding: 2px 8px 2px 0px;
```

Comment:
![image](https://user-images.githubusercontent.com/1561200/107637296-323d1b80-6c6e-11eb-816d-23c253cb2213.png)
Badge is off